[2022-08-19 01:43:00,594] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: docker_dag.print_current_date manual__2022-08-19T01:42:56.232476+00:00 [queued]>
[2022-08-19 01:43:00,610] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: docker_dag.print_current_date manual__2022-08-19T01:42:56.232476+00:00 [queued]>
[2022-08-19 01:43:00,611] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 01:43:00,612] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-08-19 01:43:00,613] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 01:43:00,639] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): print_current_date> on 2022-08-19 01:42:56.232476+00:00
[2022-08-19 01:43:00,648] {standard_task_runner.py:52} INFO - Started process 128 to run task
[2022-08-19 01:43:00,653] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'docker_dag', 'print_current_date', 'manual__2022-08-19T01:42:56.232476+00:00', '--job-id', '625', '--raw', '--subdir', 'DAGS_FOLDER/docker_dag.py', '--cfg-path', '/tmp/tmpeup6_qlm', '--error-file', '/tmp/tmpch2qv6oh']
[2022-08-19 01:43:00,655] {standard_task_runner.py:80} INFO - Job 625: Subtask print_current_date
[2022-08-19 01:43:00,749] {task_command.py:369} INFO - Running <TaskInstance: docker_dag.print_current_date manual__2022-08-19T01:42:56.232476+00:00 [running]> on host f8f5f5d21c1c
[2022-08-19 01:43:00,881] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=docker_dag
AIRFLOW_CTX_TASK_ID=print_current_date
AIRFLOW_CTX_EXECUTION_DATE=2022-08-19T01:42:56.232476+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-19T01:42:56.232476+00:00
[2022-08-19 01:43:00,885] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-19 01:43:00,887] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-08-19 01:43:00,925] {subprocess.py:85} INFO - Output:
[2022-08-19 01:43:00,939] {subprocess.py:92} INFO - Fri Aug 19 01:43:00 UTC 2022
[2022-08-19 01:43:00,940] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-19 01:43:01,088] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=docker_dag, task_id=print_current_date, execution_date=20220819T014256, start_date=20220819T014300, end_date=20220819T014301
[2022-08-19 01:43:01,194] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 01:43:01,275] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
